### DISCUSSION TITLE
168

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#129] [General]
# 潮流周刊第168期 - 想去现场

封面图来源于 22 年西湖音乐节的一个现场，氛围很好，看到这个红色和黑色的配合很有感觉，貌似快一年没有去现场了，期待周五可以抢到南京市民李先生在日本巡演的票。

https://weekly.tw93.fun/posts/168-%E6%83%B3%E5%8E%BB%E7%8E%B0%E5%9C%BA

**Comment by tw93** [comment]
有没有发现潮流周刊加载加快了，甚至你每次访问可以省掉和原来一般的流量了，哈哈。

**Reply by shellGrace** [reply]
是不是因为调整了图片大小呀

**Reply by tw93** [reply]
哈哈 图片适应、然后甚至做了预加载，比如说你鼠标放到链接上，会在你点击前提前加载内容

**Comment by jingsan0801** [comment]
如何写计算机领域的论文的 url 错了，最后少了一个 l

**Reply by tw93** [reply]
谢谢反馈

**Comment by Beeta** [comment]
最后这个视频，好听！

**Reply by tw93** [reply]
哈哈哈我也觉得

**Comment by davidchen27** [comment]
文章存图片用的是支付宝小程序的云托管服务么

**Comment by Vainycos** [comment]
每期都看，推荐的资源都很实用

**Reply by tw93** [reply]
谢谢喜欢
